### DISCUSSION TITLE
Help with download and installation

### DISCUSSION THREAD

**Discussion by rodymiester** [thejoabo/virtualfisher-bot#65] [General]
I'm a little new to doing this type of stuff and I am currently struggling to download the bot i was wondering if anyone could make a video on how to download it correctly. Thank you.

**Comment by thejoabo** [comment]
Sure,
Fastest installation: https://www.youtube.com/watch?v=PJ0o2FruocU 
Simple usage: https://www.youtube.com/watch?v=8_XH7230Tfs
